src/SFA.DAS.Assessor.Functions/Infrastructure/AccessorHttpClient.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Net.Http;
+
+namespace SFA.DAS.Assessor.Functions.Infrastructure
+{
+    public class AssessorHttpClient : HttpClient
+    {
+    }
```

Comment:
What's the point in this class?

Reply:
When I started to configure the HttpClient such that all instances were configured to be specifically connected to the Assessor API - it didn't seem quite right - so I put that in - I agree it does look odd though....